modules/psap-node-feature-discovery-using-topology-updater.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_content-type: PROCEDURE
+[id="using-the-nfd-topology-updater_{context}"]
+= Using the NFD Topology Updater
+
+The Node Feature Discovery (NFD) Topology Updater is a daemon responsible for examining allocated resources on a worker node. It accounts for resources available to be allocated to new pod on a per-zone basis, where a zone can be a Non-Uniform Memory Access (NUMA) node. The NFD Topology Updater communicates the information to nfd-master, which creates a `NodeResourceTopology` custom resource corresponding to all the nodes in the cluster. One instance of the NFD Topology Updater runs on each node of the cluster.
+
+.Procedure
```

Comment:
Just a general thought, that this procedure is about using the topology updater, but the procedure really only shows you how to get the help for it. Just wondering if it would be more helpful if the example you provided was an example of how you actually run it to do something, and then can just say to use the -help flag to find out more info on the other options of how you can run it?